dist/tsc/sponsorclient.d.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers, FetchRequest, JsonRpcApiProviderOptions, Networkish } from 'ethers';
+export declare enum WhitelistType {
+    FromAccountWhitelist = "FromAccountWhitelist",
+    ToAccountWhitelist = "ToAccountWhitelist",
+    ContractMethodSigWhitelist = "ContractMethodSigWhitelist",
+    BEP20ReceiverWhiteList = "BEP20ReceiverWhiteList"
+}
+export type WhitelistArgs = {
+    PolicyUUID: string;
+    WhitelistType: WhitelistType;
+    Values: string[];
+};
+export type GetWhitelistArgs = {
+    PolicyUUID: string;
+    WhitelistType: WhitelistType;
+    Offset: number;
+    Limit: number;
+};
+export declare class SponsorClient extends ethers.JsonRpcProvider {
+    constructor(url?: string | FetchRequest, network?: Networkish, options?: JsonRpcApiProviderOptions);
+    addToWhitelist(params: WhitelistArgs): Promise<boolean>;
+    removeFromWhitelist(params: WhitelistArgs): Promise<boolean>;
+    emptyWhitelist(params: WhitelistArgs): Promise<boolean>;
```

Comment:
I guess emptyWhitelist should not use `WhitelistArgs` as it dont have the `Values`

Reply:
fixed